consensus/chronology/chronology_test.go

```diff
@@ -285,7 +300,8 @@ func TestChronology_StartRoundShouldUpdateRoundWhenCurrentRoundIsFinished(t *tes
 	chr, _ := chronology.NewChronology(
 		syncTimerMock.CurrentTime(),
 		rounderMock,
-		syncTimerMock)
+		syncTimerMock,
+		&mock.AppStatusHandlerMock{})
```

Comment:
should also test somewhere in this file that a call to SetInt64Value was done

Reply:
done